go.mod

```diff
@@ -1,7 +1,6 @@
 module github.com/tektoncd/plumbing
 
-go 1.22
-toolchain go1.23.5
+go 1.23.0
```

Comment:
Any special reason for 1.23.0 and not 1.23.5?

Reply:
or 1.23 without the `.0` 😝

Reply:
No, there is no specific reason.
I just checked the [PR](https://github.com/tektoncd/plumbing/pull/2396) which added `toolchain go1.23.5` but the minimum required go version by the dependency was `go 1.23.0`

Reply:
what's the practice that we should follow?

Reply:
Thanks - I think the latest patch version within a version would be good, or simply 1.23 as @vdemeester said

Reply:
Sure 👍